samples/native-lib/README.md

```diff
@@ -47,7 +47,7 @@ will be generated.
 
 ```bash
 cd build
-./iwasm --native-lib=libtest_add.so --native-lib=libtest_sqrt.so wasm-app/test.wasm
+./iwasm --native-lib=./libtest_add.so --native-lib=./libtest_sqrt.so wasm-app/test.wasm
```

Comment:
i picked this commit in https://github.com/bytecodealliance/wasm-micro-runtime/pull/1643 as it conflicts otherwise.